From 2c7266a8d6eff9354be6ed7a662d77d555b516f2 Mon Sep 17 00:00:00 2001
From: beanstown106 <nbenis106@gmail.com>
Date: Fri, 23 Oct 2015 21:07:35 -0400
Subject: [PATCH 20/27] Settings: Add toggle for Lockscreen Media Art [2/3]

* Allows to choose whether or not to show album art/media art on lockscreen

Change-Id: Ieea583a0e42c317e1651ecaba66ac07c02934250
---
 res/values/cm_strings.xml        | 3 +++
 res/xml/screen_lock_settings.xml | 5 +++++
 2 files changed, 8 insertions(+)

diff --git a/res/values/cm_strings.xml b/res/values/cm_strings.xml
index 9267e88b96..93baafeb79 100644
--- a/res/values/cm_strings.xml
+++ b/res/values/cm_strings.xml
@@ -42,6 +42,9 @@
     <string name="wake_when_plugged_or_unplugged_title">Wake on plug</string>
     <string name="wake_when_plugged_or_unplugged_summary">Turn the screen on when connecting or disconnecting a power source</string>
 
+    <!-- Lock screen cover art -->
+    <string name="lockscreen_media_art_title">Display media cover art</string>
+
     <!-- Lock screen visualizer -->
     <string name="lockscreen_visualizer_title">Display music visualizer</string>
 
diff --git a/res/xml/screen_lock_settings.xml b/res/xml/screen_lock_settings.xml
index 1cb6adf84c..5f39c4dd35 100644
--- a/res/xml/screen_lock_settings.xml
+++ b/res/xml/screen_lock_settings.xml
@@ -48,6 +48,11 @@
         android:title="@string/unlock_scramble_pin_layout_title"
         android:summary="@string/unlock_scramble_pin_layout_summary" />
 
+    <lineageos.preference.LineageSecureSettingSwitchPreference
+        android:key="lockscreen_media_metadata"
+        android:title="@string/lockscreen_media_art_title"
+        android:defaultValue="true" />
+
     <lineageos.preference.LineageSecureSettingSwitchPreference
         android:key="lockscreen_visualizer"
         android:title="@string/lockscreen_visualizer_title"
-- 
2.17.1

